src/interpolations.jl

```diff
@@ -15,6 +15,7 @@ const FloatColorant{T<:AbstractFloat} = Colorant{T}
 @inline _make_compatible(T, fill) = fill
 @inline _make_compatible(::Type{T}, fill::Number) where {T} = T(fill)
 
+Interpolations.tweight(A::AbstractArray{C}) where C<:FixedPoint = C
```

Comment:
I use `C` for `Colorant` and `T` for numbers

Reply:
Of course!